consensus/misc/kip71.go

```diff
@@ -51,8 +51,13 @@ func NextBlockBaseFee(parentHeader *types.Header, config *params.ChainConfig) *b
 	if parentGasUsed == gasTarget {
 		return new(big.Int).Set(parentHeader.BaseFee)
 	} else if parentGasUsed > gasTarget {
+		// shortcut. If parentBaseFee is already reached upperbound, do not calculate.
+		if parentBaseFee.Cmp(upperBoundBaseFee) >= 0 {
```

Comment:
```suggestion
		if parentBaseFee.Cmp(upperBoundBaseFee) == 0 {
```

Reply:
@bobpkr Thanks for the suggestion. But there is a case that upperBoundBaseFee is updated by governance mechanism. In this case, parentBaseFee can be higher than upperBoundBaseFee.